index.html

```diff
@@ -1,16 +1,105 @@
-<!DOCTYPE html>
-<html lang="en">
+        <!DOCTYPE html>
+        <html lang="en">
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
-</head>
+        <head>
+            <meta charset="UTF-8">
+            <link rel="stylesheet" href="style.css">
+            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+            <meta name="viewport" content="width=device-width, initial-scale=1.0">
+            <title>Cake website</title>
+            <meta http-equiv="X-UA-Compatible" content="ie=edge">
+            <title>Responsive Cake webpage</title>
+            
+        
+            <!-- Add a link to your css file here -->
+        </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+        <body>
+        <main>
+        <!--navigation and logo-->
+        <nav class="upper-navigation"> 
+            <header><center>Welcome to Sweet cakes!</center></header>
+            <ul>
+                <li>Home</li>
+                <li>Cakes</li>
+                <li>Ordering</li>
+                <li>Lessons</li>
+                <li>About</li>
+            </ul>
+        
+        </nav>
+        <img class="cakeLogo" src="images/cake-logo.jpg" alt="logo" width="150px" height="150px">
+    
+        <!--Hello information and image carousel-->
+            <article class="intro">
+                <h1>Hello</h1>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                <p>is simply dummy text of the printing and typesetting industry.</p>
+                
+            </article>
 
-</html>
\ No newline at end of file
+            <section class="carousel">
+                    <!--making the image width 100% to make it responsive-->
+            <ul>
+                    
+                <li>
+                    <div class="carousel-container">
+                        <img src="images/cake.jpg" alt="Snow" style="width:100%">
```

Comment:
Remeber to not use inline styles within a html element - instead make use of css classes